gnomad/utils/vcf.py

```diff
@@ -796,6 +876,85 @@ def make_info_dict(
             },
         }
         info_dict.update(popmax_dict)
+    elif grpmax:
+        grpmax_dict = {
+            f"{prefix}grpmax{suffix}": {
+                "Number": "A",
+                "Description": (
+                    "Genetic ancestry group with the maximum allele"
+                    f" frequency{description_text}"
+                ),
+            },
+            f"{prefix}AC{label_delimiter}grpmax{suffix}": {
+                "Number": "A",
+                "Description": (
+                    "Allele count in the genetic ancestry group with the maximum allele"
+                    f" frequency{description_text}"
+                ),
+            },
+            f"{prefix}AN{label_delimiter}grpmax{suffix}": {
+                "Number": "A",
+                "Description": (
+                    "Total number of alleles in the genetic ancestry group with the"
+                    f" maximum allele frequency{description_text}"
+                ),
+            },
+            f"{prefix}AF{label_delimiter}grpmax{suffix}": {
+                "Number": "A",
+                "Description": (
+                    "Maximum allele frequency across genetic ancestry"
+                    f" groups{description_text}"
+                ),
+            },
+            f"{prefix}nhomalt{label_delimiter}grpmax{suffix}": {
+                "Number": "A",
+                "Description": (
+                    "Count of homozygous individuals in the genetic ancestry group"
+                    f" with the maximum allele frequency{description_text}"
+                ),
+            },
+            f"{prefix}faf95{label_delimiter}grpmax{suffix}": {
```

Comment:
i might be misremembering but i thought we dropped this?

Reply:
We did but v3 has it so I'm leaving it in so not to break that pipeline. I see this as more of a collection to pick from and modify rather than what exists in the current version